core/src/main/java/org/apache/calcite/rel/externalize/RelJson.java

```diff
@@ -475,8 +488,10 @@ RexNode toRex(RelInput relInput, Object o) {
             physical = false;
           }
           final boolean distinct = (Boolean) map.get("distinct");
-          return rexBuilder.makeOver(type, operator, rexOperands, partitionKeys,
-              ImmutableList.copyOf(orderKeys), lowerBound, upperBound, physical,
+          return rexBuilder.makeOver(type, operator, rexOperands,
+              ImmutableList.copyOf(partitionKeys),
+              ImmutableList.copyOf(orderKeys),
```

Comment:
You don't need to make a immutable copy, the RexWindow constructor make do it. orderKeys has it because of improper API definition, I guess.

Reply:
OK. updated.

Reply:
OK. updated